reactive/zookeeper.py

```diff
@@ -93,4 +100,5 @@ def serve_client(client):
     config = DistConfig()
     port = config.port('zookeeper')
     rest_port = config.port('zookeeper-rest')
-    client.send_port(port, rest_port)
+    host = get_ip_for_interface(hookenv.config().get('network_interface'))
```

Comment:
In the default case the 'network_interface' is empty and we get this exception: http://pastebin.ubuntu.com/23136576/